packages/graphql/tests/integration/directives/authorization/roles.int.test.ts

```diff
@@ -698,92 +611,6 @@ describe("auth/roles", () => {
 
             expect((gqlResult.errors as any[])[0].message).toBe("Forbidden");
         });
-
-        test("should throw if missing role on nested disconnect", async () => {
-            const typeDefs = `
-                type JWTPayload @jwt {
-                    roles: [String!]!
-                }
-
-                type ${typeComment} @node {
-                    id: String
-                    content: String
-                    post: ${typePost}! @relationship(type: "HAS_COMMENT", direction: IN)
-                }
-
-                type ${typePost} @node {
-                    id: String
-                    content: String
-                    creator: ${typeUser}! @relationship(type: "HAS_POST", direction: IN)
```

Comment:
Same as above, is there is any reason to delete this test rather than change the `creator` type